src/modules/user/services/follow.service.ts

```diff
@@ -0,0 +1,63 @@
+import { ConflictException, Injectable } from '@nestjs/common';
+import { EErrorMessage } from 'src/common/enums/error-message.enum';
+import { FollowRepository } from 'src/modules/repository/services/followe.repository';
+import { UserService } from 'src/modules/user/services/user.service';
+
+@Injectable()
+export class FollowService {
+  constructor(
+    private readonly followRepository: FollowRepository,
+    private readonly userService: UserService,
+  ) {}
+
+  public async follow(
+    follower_id: string,
+    follow_target_id: string,
+  ): Promise<void> {
+    if (follow_target_id === follower_id) {
+      throw new ConflictException(EErrorMessage.FOLLOW_YOURSELF);
+    }
+
+    const entity = await this.userService.isUserExist(follow_target_id);
+    const isFollow = await this.isFollowing(follower_id, entity.id);
+
+    if (isFollow) {
+      throw new ConflictException(EErrorMessage.ALREADY_FOLLOW);
+    }
+
+    await this.followRepository.save(
```

Comment:
maybe just do save(data)

Reply:
save nthod accept entity instance not a simple object, create - creates a new instance of Follow entity